src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -257,6 +257,9 @@ internal sealed partial class ConsoleHost
                         s_theConsoleHost.UI.WriteWarningLine(preStartWarning);
                     }
 
+                    // Send startup telemetry for ConsoleHost startup
+                    ApplicationInsightsTelemetry.SendPSCoreStartupTelemetry();
```

Comment:
i did some checking earlier (it was about 30ms), but will validate again since the code has changed to using a semaphore file. I will include that in the comments of the next push 

---
In reply to: [112802910](https://github.com/PowerShell/PowerShell/pull/3620#discussion_r112802910) [](ancestors = 112802910)